rclpy/rclpy/impl/implementation_singleton.py

```diff
@@ -28,5 +28,17 @@
 
 import importlib
 
-
-rclpy_implementation = importlib.import_module('._rclpy', package='rclpy')
+module_name = '._rclpy'
+
+try:
+    rclpy_implementation = importlib.import_module(module_name, package='rclpy')
+except ImportError as e:
+    import os
+    if os.path.isfile(e.path):
+        import sys
+        print(
```

Comment:
Use [warnings module](https://docs.python.org/3.5/library/warnings.html) instead of printing? I think it would allow an advanced user to suppress the output, though I can't think of a use case for that.